routes.js

```diff
@@ -1,24 +1,84 @@
 const express = require('express')
 const router = express.Router()
 
+const artists = require('./data/artists.json')
+const albums = require('./data/albums.json')
+const songs = require('./data/songs.json')
+
+function convertToMinutes(time) {
+  let minutes = Math.floor(time / 60)
+  let seconds = time - minutes * 60
+  if (seconds < 10) {
+    seconds = "0"+seconds
+  }
+  return minutes + ':' + seconds
+}
+
+const albumsWithArtists = albums.map((album) => {
```

Comment:
I'm seeing that you are using arrow functions but in your router functions they are not arrow functions- you could just choose one for consistency :)